test/commands/context.test.ts

```diff
@@ -82,3 +108,121 @@ describe('config', () => {
       });
   });
 });
+
+describe('config:context, negative scenario', () => {  
+  beforeAll(() => {
+    // Any context file needs to be created before starting test suite,
+    // otherwise a totally legitimate context file will be created automatically
+    // by `addContext()`.
+    testHelper.createDummyContextFileWrong('');
+  });
+
+  afterAll(() => {
+    testHelper.deleteDummyContextFile();
+  });
+
+  describe('config:context:add', () => {
+    testHelper.deleteDummyContextFile();
+    testHelper.createDummyContextFileWrong('');
+    test
+      .stderr()
+      .stdout()
+      .command(['config:context:add', 'home', './test/specification.yml'])
+      .it(
+        'should throw error on empty file saying that context file has wrong format.',
+        (ctx, done) => {
+          expect(ctx.stdout).toEqual('');
+          expect(ctx.stderr).toEqual(
+            `ContextError: Context file "${CONTEXT_FILE_PATH}" has wrong format.\n`
```

Comment:
I like we are testing it, but we also should provide some hint on how to fix it. Like `Please check your context's configuration. The file must be a valid JSON with LOREM IPSUM` or we need a follow up issue with 2 options:
- properly document structure and link error to the document
- provide JSON Schema for the config and validate and show validation errors